projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -1727,6 +1804,9 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         return null;
     }
 
+    @ViewChild('dragIndicatorIconBase', { read: TemplateRef })
+    public dragIndicatorIconBase: TemplateRef<any>;
```

Comment:
This could be marked as `@hidden @internal` as it's not really of any use in the public API